src/solutions/data-integration.njk

```diff
@@ -54,34 +53,25 @@ description:
             </div>
         </div>
         <h2 class="mb-20 m-auto">Why Node-RED is the<br>Data Integration Solution</h2>
-        <div class="grid px-6 grid-cols-1 md:grid-cols-2 gap-8 sm:gap-16 mb-16">
+        <div class="grid px-6 grid-cols-1 gap-8 mb-16">
             <div class="text-left self-start">
                 <h3><span class="text-red-600">Accessible</span> for Anyone</h3>
                 <div>
                     <p>
-                        Node-RED’s low code visual environment makes it easy for non-programmers to integrate data from different sources without needing to know how to code.
-                    </p>
-                    <p>
-                        Now, departments can work on their own data integration projects without requiring support from IT
+                        Node-RED’s low code visual environment makes it easy for non-programmers to integrate data from different sources without needing to know how to code. This enables all kinds of departments to work on their own data integration projects without requiring support from IT.
                     </p>
                 </div>
             </div>
-            <div class="text-left self-start">
-                <h3>Support for <span class="text-red-600">Different Protocols</span></h3>
-                <p>
-                    FlowFuse makes it possible for teams to collaborate on the same Node-RED flows. Team members can now share the same Node-RED instances to make for quicker and more efficient development of your Node-RED application.
-                </p>
-            </div>
             <div class="text-left self-start">
                 <h3>Real-Time <span class="text-red-600">Data Visualisation</span></h3>
                 <p>
-                    Node-RED makes it easy to create real-time data visualizations and dashboards. Data in real-time can be filtered, transformed and analyzed through a Node-RED flow.
+                    Node-RED makes it easy to create real-time data visualizations and dashboards. This data can be filtered, transformed and analyzed through a Node-RED flow to
```

Comment:
We need to decide how to continue this phrase, check the document [here](https://docs.google.com/document/d/1u12aVb-8nRzrPCacYnIAIY-95-4RwVA1Tg8010YrEhA/edit?disco=AAAA91PlOn8).